File: 백준/Bronze/5543. 상근날드/상근날드.cc
#include <iostream>

using namespace std;

int main(){ ios_base::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
    
    int price[5];
    int hamb = 2000, drink = 2000;
    
    for(int i = 0; i < 3; i++){
        cin >> price[i];
        if(price[i] < hamb){
            hamb = price[i];
        }
    }
    for(int i = 3; i < 5; i++){
        cin >> price[i];
        if(price[i] < drink){
            drink = price[i];
        }
    }
    
    cout << (hamb + drink) - 50;

}

File: 백준/Bronze/5543. 상근날드/상근날드.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] price = new int[5];
        int hamb = 2000;
        int drink = 2000;
        for(int i = 0; i < 3; i++){
            price[i] = sc.nextInt();
            if(price[i] < hamb){
                hamb = price[i];
            }
        }
        for(int i = 3; i < 5; i++){
            price[i] = sc.nextInt();
            if(price[i] < drink){
                drink = price[i];
            }
        }
        System.out.println((hamb + drink) - 50);

        
    }
}